node-fs-writefile/write-rand.js

```diff
@@ -0,0 +1,11 @@
+const data = new Uint8Array(Buffer.from(Math.random().toString()));
+
+const fs = require('fs');
+
+fs.writeFile('../rc0122-code-solutions/node-fs-writefile/random.txt', data, callBack);
```

Comment:
Your Node script should be run from within the `node-fs-writefile/` directory, and your paths should reflect that.